src/queries/queries.js

```diff
@@ -8,8 +8,8 @@ export const buildGeoJSONQueryByAccountId = accountId => ({
     .where({
       'cozyMetadata.sourceAccount': accountId
     })
-    .sortBy([{ 'cozyMetadata.sourceAccount': 'desc' }, { startDate: 'desc' }])
     .indexFields(['cozyMetadata.sourceAccount', 'startDate'])
+    .sortBy([{ 'cozyMetadata.sourceAccount': 'desc' }, { startDate: 'desc' }])
```

Comment:
why this change? 

I think we should add a missing test to be sure the bug is fixed

Reply:
Aucun bug, juste une histoire de warning en console 👍